content/harvester/_index.md

```diff
@@ -0,0 +1,28 @@
+# Testing CKAN harvesting of FAIR data points?
```

Comment:
Please confirm with a different file, but I believe you are missing the title metadata.